src/components/IOUConfirmationList.js

```diff
@@ -224,6 +224,14 @@ class IOUConfirmationList extends Component {
     }
 
     render() {
+        const buttonText = this.props.translate(
+            this.props.hasMultipleParticipants ? 'common.split' : 'iou.request', {
+                amount: this.props.numberFormat(
+                    this.props.iouAmount,
+                    {style: 'currency', currency: this.props.selectedCurrency},
+                ),
+            },
+        );
```

Comment:
We've got a condition, translation, and number format all in one chunk. Maybe separating out `amount` would simplify this statement a bit.

Reply:
Looking at it like this, I'm not sure it offers too much of an improvement 🤷 

![Screen Shot 2021-05-19 at 10 27 49 PM](https://user-images.githubusercontent.com/3885503/118879814-76852b00-b8f1-11eb-9c57-f1c114cbc388.png)